### ISSUE TITLE
shave size by removing exceptions support

### ISSUE THREAD

**Issue by anholt** [anholt/libepoxy#206]
Noticed while looking at someone else's CFLAGS, might be particularly promising for us:

https://www.chromium.org/chromium-os/build/c-exception-support

**Comment by nwnk** [comment]
I think that would break passing a callback to `glDebugMessageCallbackARB` if said passed function (called C++ code that) threw an exception.

**Comment by anholt** [comment]
Yeah, I had that thought too, but that seemed like a reasonable tradeoff.
